packages/expo-updates/scripts/create-manifest-ios.sh

```diff
@@ -13,6 +13,13 @@ elif [[ "$CONFIGURATION" == *Debug* ]]; then
   fi
 fi
 
+# ref: https://github.com/facebook/react-native/blob/c974cbff04a8d90ac0f856dbada3fc5a75c75b49/scripts/react-native-xcode.sh#L59-L65
+EXPO_UPDATES_PACKAGE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+
+# Keeps the following `$$EXPO_MODULE_VENDOR_SCRIPTS$$`` comment and
+# `expo-module-scripts' will replace the comment line with the content of `source-login-scripts.sh` during npm prepack
+source "$EXPO_UPDATES_PACKAGE_DIR/../../tools/source-login-scripts.sh" # $$EXPO_MODULE_VENDOR_SCRIPTS$$ '../../../tools/source-login-scripts.sh'
```

Comment:
Dynamically vendoring the script seems fragile. I think it would be better just to copy/paste it and commit the vendored scripts to the repo (Git will deduplicate the files in its index).

Reply:
i agree that and what's the "vendoring" in your thought? the current work is exactly copy/paste the `source-login-scripts.sh` into expo-constants package: https://github.com/expo/expo/blob/456f3557cc02dd9350af8d66f0738d90c4074fb4/packages/expo-constants/scripts/get-app-config-ios.sh#L11

Reply:
I have two ideas:

- Vendoring + committing the script to Git: Copy and paste source-login-scripts.sh into the packages that need it. If we need to, we could make `expo-module prepare` check if `scripts/source-login-scripts.sh` is defined and contains the string `# @autogenerated`. If so, it would copy the latest version of source-login-scripts.sh into the package, similar to how tsconfig.json, babel.config.js, etc. are copied.
- Depending on the the script as an npm dependency: add source-login-scripts@^1.0.0 to package.json and source `$script_dir/../../node_modules/source-login-scripts/source-login-scripts.sh`. However, this technique breaks with Yarn workspaces since the source-login-scripts package might be hoisted into the root directory.

Reply:
i like the option 1 because it won't break workspace support and easier to maintain. i'll update the pr accordingly. thanks for your advice 😁